### PR TITLE
[BUGFIX] Outname fix

### LINKED ISSUE #79: `FILE_INFO` and `PROCESS_INFO` `outname` aren't initialized.
Because they are currently initialized inside `next_finfo(3)` and `next_pinfo(3)`. Problem is that these two are called only when the corresponding arrays are out of memory.

### PR DESCRIPTION
**Pull request by fvalasiad**
Closes #79

### CODE REVIEW

File: src/tracer.c [hunk 1]
```diff
@@ -90,7 +88,7 @@ next_finfo(PROCESS_INFO *pi)
 FILE_INFO *
 finfo_at(PROCESS_INFO *pi, int index)
 {
-    if (index == pinfo->finfo_size) {
+    if (index > pinfo->numfinfo) {
```
Comment (zvr) [inline src/tracer.c:91]: Is this correct? (the case where it's equal, I mean)

Comment (fvalasiad) [reply src/tracer.c:91]: @zvr isn't `numfinfo` pointing to the last element in the array? In that case if `index` is equal to `pinfo->numfinfo` then the element already exists and thus no need to construct it. Am i missing something?

Comment (zvr) [reply src/tracer.c:91]: In C, array size 4 means 0,1,2,3.   You definitely cannot do `base+4*size` and expect to get a valid member.

I didn't check what `numfinfo` is now, but at my first write the `_size` variables had the count (e.g. 4).

Comment (fvalasiad) [reply src/tracer.c]: @zvr This should work.

**Pull request merged**